packages/cxl-lumo-styles/scss/color.scss

```diff
@@ -8,6 +8,7 @@ html,
   --lumo-primary-color: hsl(355.8, 74.7%, 48%);
   --lumo-primary-color-2pct: hsla(355.8, 74.7%, 48%, 0.02);
   --lumo-primary-color-10pct: hsla(355.8, 74.7%, 48%, 0.1);
+  --lumo-primary-color-20pct: hsla(355.8, 74.7%, 48%, 0.2);
```

Comment:
Isn't it globally available from upstream package?

Reply:
This variable didn't work otherwise